### DISCUSSION TITLE
TypeError: Cannot read properties of undefined

### DISCUSSION THREAD

**Discussion by kamleshpaliwal89** [serverless/serverless#10804] [Q&A]
Facing the below error when doing
serverless deploy --stage local

✖ Stack localstack-lambda-local failed to deploy (0s)
Environment: darwin, node 16.10.0, framework 3.7.1 (local) 3.7.1v (global), plugin 6.1.5, SDK 4.3.2
Credentials: Local, "default" profile
Docs:        docs.serverless.com
Support:     forum.serverless.com
Bugs:        github.com/serverless/serverless/issues

Error:
TypeError: Cannot read properties of undefined (reading 'console')
 at aws:deploy:deploy:checkForChanges

Can someone please help here?

**Comment by pgrzesik** [comment]
Hello @kamleshpaliwal89 - this looks like a bug with that is reported here: https://github.com/localstack/serverless-localstack/issues/180 

Please continue the discussion in the plugin's repository as it will have to be fixed on the plugin side.
